heartbeat/pgsql

```diff
@@ -1479,7 +1479,7 @@ set_async_mode_all() {
 }
 
 set_async_mode() {
-    cat $REP_MODE_CONF |  grep -q -e "[,' ]$1[,' ]"
+    cat $REP_MODE_CONF |  grep -q -e "\"$1\""
```

Comment:
$1 is should set hostname. I think [', ] are invalid hostname characters, so there is probably no case that these characters are included inside " ".
But I think the pattern `grep -E "(\"$1\")|([,' ]$1[,' ])"` is good for backward compatible.
I confirmed with 2 hostname pattern of slave node with the pattern `grep -E "(\"$1\")|([,' ]$1[,' ])"`
1. valid SQL identifier
pm02
2. invalid SQL identifier
pm02.test

This pattern works fine both.